.rubocop.yml

```diff
@@ -1,31 +1,50 @@
+
 AllCops:
   Exclude:
     - "bin/*"
     - "db/**/*"
     - "config/**/*"
     - "Guardfile"
     - "Rakefile"
-    - "node_modules/**/*"
   DisplayCopNames: true
+  DisabledByDefault: true
```

Comment:
You have excluded everything and disabled the cops
It is should not be like this.